packages/universal-components/src/Button/ButtonInner.js

```diff
@@ -78,24 +81,40 @@ export default function ButtonInner({
         circled && styleSheet.buttonCircled,
         sizeTheme(size, leftIcon != null, rightIcon != null).buttonSizeWrapper,
         style,
+        isLoading && styleSheet.loadingState,
       ]}
       testID={testID}
     >
-      <View style={layout.row}>
-        {leftIcon != null && <View style={rightSpace}>{leftIcon}</View>}
-        {children ||
-          (label != null && (
-            <ButtonTitle text={label} type={type} size={size} />
-          ))}
-      </View>
-      <View style={layout.row}>
-        {sublabel != null && (
+      {isLoading ? (
+        <View style={{ alignSelf: 'center' }}>
+          <Loader
+            size={size === 'large' ? 'large' : 'small'}
+            color={
+              type === 'secondary' || type === 'google'
+                ? defaultTokens.paletteProductNormal
+                : defaultTokens.paletteWhite
+            }
+          />
+        </View>
+      ) : (
+        <>
           <View style={layout.row}>
-            <ButtonTitle text={sublabel} type={type} variant="sublabel" />
+            {leftIcon != null && <View style={rightSpace}>{leftIcon}</View>}
+            {children ||
```

Comment:
`children != null`

Reply:
Where do you want to add this condition?

Reply:
Oh, I think I misread it, nevermind.